src/data-workspace/category-combo-table/category-combo-table.js

```diff
@@ -174,6 +184,7 @@ export const CategoryComboTable = ({
                                 <DataEntryField
                                     dataElement={de}
                                     categoryOptionCombo={coc}
+                                    disabled={isGreyedField(de, coc)}
                                 />
                             </DataEntryCell>
```

Comment:
What do you think about just rendering a padding cell here if this is a greyed cell? It would save rendering a disabled input:
```js
isGreyedField(de, coc) ? <PaddingCell key={coc.id} /> : <DataEntryCell /* ...etc... */ />
```

Reply:
Okay! I think that's a funny design haha, but I will defer to how the old app works 👍